src/mito2/src/sst/file.rs

```diff
@@ -125,7 +125,7 @@ impl FileMeta {
 }
 
 /// Handle to a SST file.
-#[derive(Clone)]
+#[derive(Clone, Serialize, Deserialize)]
 pub struct FileHandle {
```

Comment:
It might not be a good idea to implement `Serialize` and `Deserialize` for `FileHandle`.